This started out as a patch to remove makeJar.csh but as I dug deeper into the
mess it's gathered other Java fixups.  When I actually get the thing straight
I will give it a proper name.
TIM
--- a/jemboss/utils/Makefile.am
+++ b/jemboss/utils/Makefile.am
@@ -5,7 +5,6 @@
 jembossutils_DATA = \
 install-jemboss-server.sh \
 keys.sh \
-makeJar.csh \
 makeJNLP.sh \
 makeFileManagerJNLP.sh \
 template.html \
--- a/jemboss/runJemboss.sh.in
+++ b/jemboss/runJemboss.sh.in
@@ -28,6 +28,8 @@
 CLASSPATH="${CLASSPATH}:${JEMBOSS_HOME}/lib/axis/saaj.jar";
 CLASSPATH="${CLASSPATH}:${JEMBOSS_HOME}/lib/axis/wsdl4j.jar";
 
+CLASSPATH="${CLASSPATH}:/usr/share/EMBOSS/data/"
+
 cd "${JEMBOSS_HOME}";
 
 # Add local to run Jemboss in 'standalone' mode:
--- a/jemboss/lib/Makefile.am
+++ b/jemboss/lib/Makefile.am
@@ -6,7 +6,6 @@
 
 jembosslib_DATA = \
 README \
-activation.jar \
 jakarta-regexp-1.2.jar \
 mail.jar \
 jalviewApplet.jar \
--- a/jemboss/runJemboss.sh
+++ b/jemboss/runJemboss.sh
@@ -11,7 +11,7 @@
 
 export CLASSPATH="";
 
-CLASSPATH="${CLASSPATH}:${JEMBOSS_HOME}/lib/activation.jar";
+#CLASSPATH="${CLASSPATH}:/usr/share/java/activation.jar";
 CLASSPATH="${CLASSPATH}:${JEMBOSS_HOME}/lib/client.jar";
 CLASSPATH="${CLASSPATH}:${JEMBOSS_HOME}/lib/jakarta-regexp-1.2.jar";
 CLASSPATH="${CLASSPATH}:${JEMBOSS_HOME}/lib/jalviewApplet.jar";
